structuredefinitions/Extension-UKCore-SampleCategory.xml

```diff
@@ -32,16 +32,16 @@
   <differential>
     <element id="Extension">
       <path value="Extension" />
-      <short value="An extension to record the state of the sample." />
-      <definition value="This extension allows the recording of the state of a sample for use within Genomics, whether it is a solid tumour, liquid tumour, normal, or germline sample." />
+      <short value="An extension to record the category of the sample." />
+      <definition value="This extension allows the recording of the category of a sample for use within Genomics, whether it is a solid tumour, liquid tumour, normal, or germline sample." />
```

Comment:
Remove reference to Solid/Liquid/Normal types, only Tumour/Germline should remain